packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -1389,84 +1332,80 @@ describe("TableFactory unit tests", () => {
 
 			// Update cell value
 			const cell =
-				treeView.root.getCell({
+				table.getCell({
 					row: "row-0",
 					column: "column-0",
 				}) ?? fail("Cell not found");
 			cell.value = "Updated value!";
 			assert.equal(eventCount, 4);
 		});
 
-		it("Responding to column list changes", () => {
-			const { treeView } = createTableTree();
-			treeView.initialize({
-				columns: [],
-				rows: [],
-			});
+		// TODO:AB#47404: Fix column removal for unhydrated table trees and re-enable in unhydrated mode.
+		if (hydrated) {
+			it("Responding to column list changes", () => {
```

Comment:
@noencke this test specifically seems to be failing in the unhydrated case because the column re-ordering operation on line 1374 is causing 2 change events to be fired instead of the 1 we see for hydrated nodes. 

Does this violate our spec? It seems like it should. I.e., there shouldn't be any behavioral differences between hydrated and unhydrated trees.

Reply:
Filed a bug here: https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/47457